test/js/ui/map.test.js

```diff
@@ -56,6 +56,11 @@ test('Map', (t) => {
         t.ok(map.keyboard.isEnabled());
         t.ok(map.scrollZoom.isEnabled());
         t.ok(map.touchZoomRotate.isEnabled());
+        t.throws(() => {
+            new Map({
+                container: 'anElementIdWhichDoesNotExistInTheDocument'
+            });
+        }, Error, 'throws on invalid map container id');
```

Comment:
I had problems with getting the object with a regex approach to work. I tried `{Error:/^Container 'anElementIdWhichDoesNotExistInTheDocument' not found$/}`, so in the end I simply used `new Error("Container 'anElementIdWhichDoesNotExistInTheDocument' not found")` which worked.

I checked that the test failed without the change to js/ui/map.js and then passed with the change in js/ui/map.js.